extmod/modujson.c

```diff
@@ -77,28 +79,43 @@ STATIC byte ujson_stream_next(ujson_stream_t *s) {
     return s->cur;
 }
 
+// We read from an object's `readinto` method in chunks larger than the json
+// parser needs to reduce the number of function calls done.
+
+#define CIRCUITPY_JSON_READ_CHUNK_SIZE 64
+
 STATIC mp_uint_t ujson_python_readinto(mp_obj_t obj, void *buf, mp_uint_t size, int *errcode) {
```

Comment:
parameter 'size' was used
```
../../extmod/modujson.c: In function ‘ujson_python_readinto’:
../../extmod/modujson.c:87:75: **error: unused parameter ‘size**’ [-Werror=unused-parameter]
   87 | STATIC mp_uint_t ujson_python_readinto(mp_obj_t obj, void *buf, mp_uint_t size, int *errcode) {
      |                                                                 ~~~~~~~~~~^~~~
cc1: all warnings being treated as errors
```

Reply:
You are right. I just pushed a `(void) size;`.